templates/js-cypress/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "cypress-module-api",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "Run Cypress via its module API",
+  "engines": {
+    "node": ">=16.0.0"
```

Comment:
You have `18.12.0` in the `.nvmrc` so maybe it could be `"node": ">=18.0.0" `?

Reply:
goood catch 👌